Makefile

```diff
@@ -23,9 +31,26 @@ types_easyjson.go: types.go
 test: types_easyjson.go
 	go test -v
 
+.PHONY: lint
+lint:
+	go vet ./...
+	golangci-lint run
+
+.PHONY: e2e-tests-env-setup
+e2e-tests-env-setup:
+	./hack/setup-e2e-env.sh
+
+.PHONY: e2e-tests-env-destroy
+e2e-tests-env-destroy:
+	./hack/destroy-e2e-env.sh
+
 .PHONY: e2e-tests
 e2e-tests: annotated-policy.wasm
-	bats e2e.bats
+	@echo WIP
+	exit 0
```

Comment:
I would at least naively check that the Rancher Project `p-testing` is present, exit 1 if not, or run `bats e2e.bats`.

Reply:
fixed, now doing testing against a recorded session